### PR TITLE
491-CI-fix-tarball-download

### PR DESCRIPTION
**Pull request by Jorge-Rodriguez**
Fixes the CI setup to pull MariaDB tarball only on MariaDB runs. Updates the download URL for MySQL tarball.

### CODE REVIEW

File: changelogs/fragments/491_fix_download_url.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+- minor_changes:
+  - setup_mysql - Update MySQL tarball URL (https://github.com/ansible-collections/community.mysql/pull/491)
```
Comment (Andersson007) [inline changelogs/fragments/491_fix_download_url.yaml]:
```suggestion
  - setup_mysql - update MySQL tarball URL (https://github.com/ansible-collections/community.mysql/pull/491).
```
to make it look consistent with other changelog entries

**Comment by laurent-indermuehle** [comment]
Oh, I'm sorry @Jorge-Rodriguez ... `setup_mysql` may be deleted soon by #490. So maybe don't work on the CI until we decide if we go with my proposition or not. I should have posted something in the discussion here in addition to my messages on Matrix.

**Comment by Andersson007** [comment]
> Oh, I'm sorry @Jorge-Rodriguez ... `setup_mysql` may be deleted soon by #490. So maybe don't work on the CI until we decide if we go with my proposition or not. I should have posted something in the discussion here in addition to my messages on Matrix.

@laurent-indermuehle I think it's OK to merge it now if you're OK. I asked @Jorge-Rodriguez to make this PR so that our contributors could run tests until your PR gets merged

**Comment by laurent-indermuehle** [comment]
@Andersson007 That's ok then, I didn't notice the CI was broken. This will not conflict because #490 delete those files.
(Sounds like you already know that my PR will not be merge soon 😅)

@Jorge-Rodriguez I've fixed a syntax issue in the changelog fragment. If it pass I'll be happy to merge. Great job for the Shell part! (I especially like  the == mariadb* ]] )

**Comment by Andersson007** [comment]
@Jorge-Rodriguez thanks for the contribution!

@laurent-indermuehle hehe, yeah, i've no idea when we have everything green in the CI overhaul PR:)

**Pull request merged**